modules/machine-config-drift-detection.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="machine-config-drift-detection_{context}"]
+= Understanding configuration drift detection
+
+There might be situations when the on-disk state of a node differs from what is configured in the machine config. This is known as _configuration drift_. For example, a cluster admin might manually modify a file, a systemd unit file, file permissions or other operating system feature that was configured through a machine config. This causes configuration drift. Configuration drift can cause problems between nodes in a Machine Config Pool or when the machine configs are updated.  
```

Comment:
> For example, a cluster admin might manually modify a file, a systemd unit file, file permissions or other operating system feature that was configured through a machine config.

It would be more accurate to omit the, "other operating system feature" in this case. We only detect config drift based upon file contents, systemd unit file contents, or file permissions. Other operating system features such as kernel arguments, SSH keys, etc. are not checked.